dist/Colordepth.js

```diff
@@ -1,4 +1,4 @@
-var calcColorDepth = function() {

```

Comment:
Pro tip, review this with hiding whitespace:

<img width="423" alt="image" src="https://user-images.githubusercontent.com/10931297/163599526-13834893-b536-43ea-ad67-2fbb61ced2c5.png">